projects/ion/src/lib/divider/divider.component.html

```diff
@@ -1,8 +1,16 @@
 <hr
+  *ngIf="!(type === 'text' && direction === 'horizontal')"
   data-testid="hr"
   [style.margin]="margin ? '0' : ''"
-  [class]="'ion-divider-' + direction + ' ion-divider-' + type"
-  [attr.data-content]="
-    direction === 'horizontal' ? (type === 'text' ? label : '') : ''
+  [class]="
+    'ion-divider-' + direction + (type === 'text' ? '' : ' ion-divider-' + type)
   "
 />
+
+<ng-container *ngIf="type === 'text' && direction === 'horizontal'">
```

Comment:
I believe that this can be a function because will not duplicated here.

Reply:
Due to performance issues caused by this, it is not good to use functions in the template

Reply:
> Due to performance issues caused by this, it is not good to use functions in the template

Nice, LGTM.